galaxy.yml

```
@@ -0,0 +1,70 @@
+---
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: "networktocode"
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: "netauto"
```

I would update this to match the name of the repo
```suggestion
name: "ntc_ansible"
```

---

@jvanderaa thoughts on the name?  I feel like I remember you mentioning a collision on naming for ntc-ansible but can't remember if that was because you were thinking `namespace: networktocode` `name: ansible` vs `name: ntc_ansible`.

---

maybe worth dropping the `ntc_` from the module names then too..

---

I thought we were going away from the `ansible` name in there. It may be worth asking a question in the Slack. Let me know if you want me to help on that @jeffkala

---

we've agree on `netauto`.